libdnf/dnf-sack.h

```diff
@@ -104,6 +104,29 @@ int          dnf_sack_evr_cmp               (DnfSack        *sack,
                                              const char     *evr2);
 const char  *dnf_sack_get_cache_dir         (DnfSack        *sack);
 DnfPackage  *dnf_sack_get_running_kernel    (DnfSack        *sack);
+/**
+ * @brief Store Map with only pkg_solvables to increase query performance
```

Comment:
gtk-doc comments should be in C file

Reply:
@ignatenkobrain What about on both places?

Reply:
@ignatenkobrain Another point of view.
1. dnf-sack.h is public header file. That means public API. Why you want hide comments into C file? If developer installs devel (header) files then he wont see the C code. It means he must install C code or documentation. But if comments are separated to another documentation, then IDE must support this documentation to be fully usable.

2. What about static inline functions defined in header file? And what about types defined in header file?

Reply:
1. gtk-doc is generated out of C files
2. But this is not static inline function. And depends which types, if those are public, they should be described in header.

https://developer.gnome.org/gtk-doc-manual/stable/